packages/number-input/src/NumberInput/NumberInput.types.ts

```diff
@@ -136,12 +136,12 @@ export interface BaseNumberInputProps
   /**
    * The message shown below the input element if the value is invalid.
    */
-  errorMessage?: string;
+  errorMessage?: React.ReactNode;
```

Comment:
Do we not have a shared type for these props?

Reply:
not currently. do you think it's worth it if it's a simpler type like `ReactNode`?